exercises/high-scores/description.md

```diff
@@ -0,0 +1,3 @@
+Manage a game player's High Score list.
+
+The high score list knows about the highest scores, the last added score and other info useful to a game player.

```

Comment:
I'd like to see more description of what the task actually is.

Reply:
Thanks @insti, for the review, much appreciated! 
How about this for the description:
```
The high score list knows about the highest scores, the last added score and other info useful to a game player.
Your task is to write methods that return the highest score from the list, the last added score, the three highest scores, and a report on the difference between the last and the highest scores. 
```

Reply:
It's a step in the right direction.

It's nice to provide a little more context and "color"

Have look at the description for exercises like `book-store` or `kindergarten-garden` or `two-bucket`